### DISCUSSION TITLE
OsgEarth-2.10.1,Missing MSVCR90.dll

### DISCUSSION THREAD

**Discussion by Atahh** [pelicanmapping/osgearth#2319] [Support Q&A]
Hello, I have been building osg earth-2.10.1, for the past couple days now. I have osg-3.6.5 already and it runs the examples and passes the test. My dependences I have for osg earth-2.10.1 are, CURL, GDAL, GEOS, and SQlite3. I build and install everything and it gives no errors, but when I try to run the test files, I get an error saying missing MSVCR90.dll. I know the file is in the WinSxS directory, but when I put the path to it I get another error saying, C run time error. Any help would be greatly appreciated.

**Comment by gwaldron** [comment]
Hard to say. I would guess that something was built with VS 2008 and is trying to find a DLL that's wither missing or corrupt. You may need to replace or download the DLL from Microsoft, or built with more modern versions of VS and your dependencies. Just a stab in the dark.

**Reply by Atahh** [reply]
I have built all the dependencies with visual studio 2019, I tried downloading the 2008 redistributle and still gives the same error, do by chance know the version for the dependencies for 2.10.1, I had Gdal 3.7, Geos 3.6.3, Curl 8.1.2. I built osg just fine and everything works.
